src/jsdocUtils.js

```diff
@@ -23,8 +23,14 @@ const getFunctionParameterNames = (functionNode : Object) : Array<string> => {
       return param.argument.name;
     }
 
+    if (param.type === 'TSParameterProperty') {
+      return functionParameterNameMapper(param.parameter);
+    }
+
     throw new Error('Unsupported function signature format.');
-  });
+  };
+
+  return _.map(functionNode.params, functionParameterNameMapper);
```

Comment:
```suggestion
  return functionNode.params.map(getParamName)
```
We are moving away from lodash.